scala-xml/src/main/scala/scalaxml/ElemInstances.scala

```diff
@@ -8,7 +8,11 @@ import scala.util.control.NonFatal
 import scala.xml._
 import scalaz.concurrent.Task
 
+import javax.xml.parsers.SAXParserFactory
+
 trait ElemInstances {
+  private val spf = SAXParserFactory.newInstance()
```

Comment:
Make this an abstract def, and implement it as `SAXParserFactory.newInstance()` in the package object?  Then anyone needing customization can extend `ElemInstances` and import that instead of the package object.  Similar to how pretty printing is made configurable in JSON.